schema.sql

```diff
@@ -9,4 +9,46 @@ CREATE TABLE animals (
   neutered BOOLEAN,
   weight_kg DECIMAL (5,2),
   PRIMARY KEY (id);
-);
\ No newline at end of file
+);
+
+ALTER TABLE animals ADD species varchar(150);
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(150),
+    age INT
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(150)
+);
+
+CREATE SEQUENCE IF NOT EXISTS animals_id_seq;
+SELECT SETVAL('animals_id_seq',(SELECT max(id) FROM animals));
+ALTER TABLE animals
+    ALTER COLUMN id
+    SET DEFAULT nextval('animals_id_seq'::regclass);
+ALTER SEQUENCE animals_id_seq OWNED BY animals.id;
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals
+    ADD COLUMN IF NOT EXISTS species_id INT;
+
+ALTER TABLE animals
+    ADD CONSTRAINT fk_species_id FOREIGN KEY (species_id) REFERENCES species(id);
+
+ALTER TABLE animals
+    ADD COLUMN IF NOT EXISTS owner_id INT;
+    
+ALTER TABLE animals
+    ADD CONSTRAINT fk_owner_id FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+Footer
+© 2023 GitHub, Inc.
+Footer navigation
+Terms
+Privacy
+Security
+Status
```

Comment:
- [ ] Please remove these lines of code as they are not SQL statements and they are not relevant to the project. If you run this file, you will get a syntax error similar to the one in the screenshot attached below:

![image](https://user-images.githubusercontent.com/77106962/235942202-294e2e57-5a86-4154-b524-cd255c3bf52e.png)

Reply:
This was just for the information, But I removed it also. 😁